array-manipulation-3/src/flatten.js

```diff
@@ -0,0 +1,10 @@
+/* exported flatten */
+// create a function to combine arrays with their sub arrays
+// create a variable to take in the array
+// create a new array and concatenate it with the values inside of the original array
+// return the new array from the function.
+
+function flatten(array) {
+  var flatten = [].concat.apply([], array);
```

Comment:
You should try to do this by thinking through the logic yourself rather than relying on this obscure JavaScript mechanic 🤓